src/main/java/com/aws/greengrass/logmanager/CloudWatchLogsUploader.java

```diff
@@ -144,10 +145,14 @@ private boolean uploadLogs(String logGroupName, String logStreamName, List<Input
             return true;
         } catch (InvalidSequenceTokenException e) {
             // Get correct token using describe
-            logger.atError()
-                    .log("Invalid token while uploading logs to {}-{}. Getting the correct sequence token.",
-                            logGroupName,
-                    logStreamName);
+            if (tryCount < MAX_RETRIES) {
+                logger.atInfo().log("Invalid token while uploading logs to {}-{}. Retrying with the expected sequence "
+                                + "token in CloudWatch response.", logGroupName, logStreamName);
+            } else {
+                logger.atError().cause(e).log("Invalid token while uploading logs to {}-{} with max retry ({}) times "
```

Comment:
I don't think that logging the cause is very necessary since this is very specifically an InvalidSequenceTokenException

Reply:
will update it.